run_dir/design/ont_flowcell.html

```diff
@@ -0,0 +1,156 @@
+{% extends 'base.html' %}
+{% block stuff %}
+{% import re %}
+<div id="page_content">
+    <h1>Flow Cell <span id="page_title">{{ flowcell.get('run_name') }}</span></h1>
+    <table class="table table-bordered narrow-headers" id="fc_info">
+      <tr class="darkth">
```

Comment:
It would be nice if the table grey identifier rows were a bit narrower?

Reply:
This will be taken into account when the single-flowcell page is further developed!